template/src/css/main.css

```diff
@@ -112,6 +112,19 @@ td.code {
   user-select: none;
 }
 
+.label-required {
```

Comment:
I think it would be better to have a class for labels, and then optional/required which is just changing the color